library/core/src/cmp.rs

```diff
@@ -634,7 +636,19 @@ impl<T: Clone> Clone for Reverse<T> {
 /// An order is a total order if it is (for all `a`, `b` and `c`):
 ///
 /// - total and asymmetric: exactly one of `a < b`, `a == b` or `a > b` is true; and
```

Comment:
FWIW I think "total" might be a confusing term to use here... "an order is a total order if it is total"?
The formal term seems to be ["Trichotomy"](https://en.wikipedia.org/wiki/Trichotomy_(mathematics)).